types/create-keyframe-animation/create-keyframe-animation-tests.ts

```diff
@@ -0,0 +1,43 @@
+import animations from "create-keyframe-animation";
```

Comment:
```suggestion
import * as animations from "create-keyframe-animation";
```
ditto (about exports)